jumanji/environments/swarms/search_and_rescue/env.py

```diff
@@ -167,9 +170,12 @@ def __init__(
             view_angle=searcher_view_angle,
         )
         self.max_steps = max_steps
-        self._viewer = viewer or SearchAndRescueViewer()
         self._target_dynamics = target_dynamics or RandomWalk(0.01)
         self.generator = generator or RandomGenerator(num_targets=20, num_searchers=10)
+        self._viewer = viewer or SearchAndRescueViewer()
+        # Needed to set environment boundaries for plots
+        if isinstance(self._viewer, SearchAndRescueViewer):
+            self._viewer.env_size = (self.generator.env_size, self.generator.env_size)
```

Comment:
Instead of this could we pass the env size into `render`/`animate`

Reply:
Yeah, was not sure the best approach, thought it might be best to not have it set in 2 places, but happy to switch out.

Reply:
Ye this just seems a bit brittle to me, eg what happens if we use another viewer for some reason.